rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/ServerResource.java

```diff
@@ -528,4 +539,66 @@ protected Response exportPersistenceData(MementoCopyMode preferredOrigin) {
         }
     }
 
+    @Override
+    public Response importPersistenceData(String persistenceExportLocation) {
```

Comment:
Any tests for this? More comments and/or named constants/vars would be good to have as well (just to understand what it does).